backend/tournesol/tests/test_api_polls.py

```diff
@@ -202,3 +203,82 @@ def test_anon_can_list_videos_filtered_by_metadata_mixed(self):
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
         self.assertEqual(resp.data["count"], 0)
         self.assertEqual(resp.data["results"], [])
+
+class EntityPollDistributorTestCase(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.poll_videos = Poll.default_poll()
+
+        user1 = User.objects.create_user(username="user1", email="test1@example.test")
+        user2 = User.objects.create_user(username="user2", email="test2@example.test")
+
+        self.entity1 = EntityFactory()
+
+        self.contributor_ratings_1 = ContributorRatingFactory(user=user1, entity=self.entity1)
+        self.contributor_ratings_2 = ContributorRatingFactory(user=user2, entity=self.entity1)
+
+    def test_basic_api_call_is_successfull(self):
+        response = self.client.get(
+            f"/polls/videos/entities/{self.entity1.uid}/criteria_scores_distributions")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
```

Comment:
This test currently would fail with a 500 instead of the expected 404

```suggestion

   def test_basic_call_missing_entity(self):
        response = self.client.get(f"/polls/videos/entities/XYZ/criteria_scores_distributions")
        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)

```